### PR TITLE
Add a LogKill event transformer

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

File: transformers/integration_tests/log_kill_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+		Expect(dbResult.Maker).To(Equal(expectedMakerID))
+		Expect(dbResult.PayGem).To(Equal(expectedPayGemID))
+		Expect(dbResult.BuyGem).To(Equal(expectedBuyGemID))
+		Expect(dbResult.PayAmt).To(Equal("97221450402889691581"))
+		Expect(dbResult.BuyAmt).To(Equal("34990000000000000001"))
+		Expect(dbResult.Timestamp).To(Equal("1583440730"))
+		Expect(dbResult.AddressID).To(Equal(expectedAddressID))
+	})
+
+	FIt("fetches and transforms a LogKill event for OASIS_MATCHING_MARKET_TWO contract", func() {
```
Comment (rmulhol) [inline transformers/integration_tests/log_kill_test.go:83]: Think we need to swap out the `FIt` to fix the build

File: transformers/events/log_kill/transformer_test.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		Expect(payGemAddressErr).NotTo(HaveOccurred())
+		buyGemAddressId, buyGemAddressErr := shared.GetOrCreateAddress(test_data.BuyGemAddress.Hex(), db)
+		Expect(buyGemAddressErr).NotTo(HaveOccurred())
+
+		expectedModel.ColumnValues[event.AddressFK] = addressId
+		expectedModel.ColumnValues[constants.MakerColumn] = makerAddressId
+		expectedModel.ColumnValues[constants.PayGemColumn] = payGemAddressId
+		expectedModel.ColumnValues[constants.BuyGemColumn] = buyGemAddressId
+
+		Expect(models).To(Equal([]event.InsertionModel{expectedModel}))
```
Comment (rmulhol) [inline transformers/events/log_kill/transformer_test.go:44]: This reads much nicer than the transformer tests I've been writing 😅 

May open up a small PR to do address ID assignments this way elsewhere 👍

File: transformers/events/log_kill/transformer.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+				event.LogFK,
+				event.AddressFK,
+				constants.OfferId,
+				constants.PairColumn,
+				constants.MakerColumn,
+				constants.PayGemColumn,
+				constants.BuyGemColumn,
+				constants.PayAmtColumn,
+				constants.BuyAmtColumn,
+				constants.TimestampColumn,
```
Comment (rmulhol) [inline transformers/events/log_kill/transformer.go:84]: ditto here 🤩

File: db/migrations/20200415162259_create_log_kill_table.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    timestamp  BIGINT,
+    UNIQUE (header_id, log_id)
+);
+
+CREATE INDEX log_kill_header_index
+    ON maker.log_kill (header_id);
+CREATE INDEX log_kill_index
+    ON maker.log_kill (log_id);
+CREATE INDEX log_kill_address_index
+    ON maker.log_kill (address_id);
```
Comment (rmulhol) [inline db/migrations/20200415162259_create_log_kill_table.sql:24]: May want indexes on the `maker`/`pay_gem`/`buy_gem` FKs as well?

File: transformers/integration_tests/log_kill_test.go (hunk 1 above)
Comment (elizabethengelman) [reply transformers/integration_tests/log_kill_test.go:83]: 🤦‍♀️ nice catch!

**Pull request merged**